Source/DafnyLanguageServer.Test/Diagnostics/DiagnosticsTest.cs

```diff
@@ -38,12 +38,36 @@ import   UnderlineComments2
 
     [Fact]
     public async Task RedundantAssumptionsGetWarnings() {
-      var path = Path.Combine(testFilesDirectory, "ProofDependencies/LSPProofDependencyTest.dfy");
-      var documentItem = CreateTestDocument(await File.ReadAllTextAsync(path), path);
+      var directory = Path.Combine(Path.GetTempPath(), Path.GetRandomFileName());
+      Directory.CreateDirectory(directory);
```

Comment:
Shouldn't this directory be deleted at the end of the test?

Reply:
That would be nicer. I've added a commit that deletes the test directories created by any of the language server tests.